code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -407,6 +407,15 @@
 		user.buckled.unbuckle_mob()
 	spawn(0)
 		var/mob/living/U = user
+		for(var/obj/item/weapon/implant/tracking/T in U)//Checking if a tracking implant is in the mob
```

Comment:
Could we not have this, and instead let holy water deplete a vampire's blood supply, if security seeks to imprison a vampire? Special-case code like this is kinda blech